src/trezor/wire.cpp

```diff
@@ -25,6 +25,13 @@
 
 #include "wire.h"
 
+#define DEBUG_TRANSFER     0
+
+#define TREZOR1_VID        0x534c
+#define TREZOR1_PID        0x0001
+#define TREZOR2_VID        0x1209
+#define TREZOR2_PID        0x53c1
```

Comment:
My understanding is we might want to also support device ID `0x53c0` for trezor T "prereleases"

Reply:
I've added the suggestions you made – all good ones I overlooked! Yes, the if (false) was from before I merged your commit with the updated protobufs, since the PasswordStateRequest was a newly added message type, which was missing from the older protobufs.

Anyway, the 0x53c0 PID isn't necessary for us – turns out that isn't a prerelease unit, but the PID their bootloader uses for a firmware-less model T (they ship without firmware and prompt you to install it during the setup procedure as a security precaution). See [here](https://github.com/trezor/python-trezor/blob/92383824351b7f129b1f1534910f8d0786ce5284/trezorlib/transport/webusb.py#L31)